internal/txemail/txemail.go

```diff
@@ -116,6 +117,16 @@ func Send(ctx context.Context, message Message) error {
 		return fmt.Errorf("invalid SMTP authentication type %q", conf.EmailSmtp.Authentication)
 	}
 
+	if conf.EmailSmtp.DisableTLS {
+		return m.SendWithTLS(
+			net.JoinHostPort(conf.EmailSmtp.Host, strconv.Itoa(conf.EmailSmtp.Port)),
+			smtpAuth,
+			&tls.Config{
+				InsecureSkipVerify: true,
+			},
+		)
+	}
+
 	return m.Send(
 		net.JoinHostPort(conf.EmailSmtp.Host, strconv.Itoa(conf.EmailSmtp.Port)),
 		smtpAuth,
```

Comment:
Just wondering, can we just do this:

```suggestion
	return m.SendWithTLS(
		net.JoinHostPort(conf.EmailSmtp.Host, strconv.Itoa(conf.EmailSmtp.Port)),
		smtpAuth,
		&tls.Config{
			InsecureSkipVerify: conf.EmailSmtp.DisableTLS,
		},
	)
```

Reply:
Good question, I'll give it a try

Reply:
We tried with @unknwon and it doesn't work because the library doesn't like passing an empty tls.Config in some cases